data/items.ts

```diff
@@ -1,3 +1,6 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import {Pokemon} from "../sim";
```

Comment:
What is this for?

Reply:
es lint errors

Reply:
I mean, you aren't using it anywhere and had to disable eslint _because_ you aren't using it. Just remove it entirely?